nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AttributesToJSON.java

```diff
@@ -79,6 +80,17 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+
+    public static final PropertyDescriptor ATTRIBUTES_REGEX = new PropertyDescriptor.Builder()
+            .name("attributes-to-json-regex")
+            .displayName("Attributes Regular Expression")
+            .description("Regular expression that will be evaluated against the flow file attributes to select "
+                    + "the matching attributes. This property can be used in combination with the attributes "
+                    + "list property.")
+            .required(false)
+            .addValidator(StandardValidators.REGULAR_EXPRESSION_VALIDATOR)
```

Comment:
What do you think about supporting Expression Language for this field? You can keep the same validator, although that doesn't allow validation against flow files, just Variable Registry properties and such. You could support regexes from flow file attributes too, but you'd need a new validator and it would recompile each time, which might not be a great idea from a performance perspective.